test/include/test.hpp

```diff
@@ -333,7 +333,7 @@ enum class color
     bg_blue    = 44,
     bg_default = 49
 };
-inline std::ostream& operator<<(std::ostream& os, const color& c)
+inline std::ostream& operator<<(std::ostream& os, [[maybe_unused]] const color& c)
```

Comment:
I would prefer a `(void)c` added under `#ifdef _WIN32`, so we can still get diagnostics for unused variable on the linux side.